src/celeritas/phys/PhysicsData.hh

```diff
@@ -242,9 +242,14 @@ struct PhysicsParamsScalars
 
     // User-configurable multiple scattering options
     real_type lambda_limit{};  //!< lambda limit
-    real_type range_factor{};  //!< range factor for e-/e+ (0.2 for muon/h)
+    real_type range_factor{};  //!< range factor for e-/e+
+    real_type muhad_range_factor{};  //!< range factor for muon/h
     real_type safety_factor{};  //!< safety factor
+    bool displaced{};  //!< lateral displacement for e-/e+
+    bool muhad_displaced{};  //!< lateral displacement for muon/h
     MscStepLimitAlgorithm step_limit_algorithm{MscStepLimitAlgorithm::size_};
+    MscStepLimitAlgorithm muhad_step_limit_algorithm{
+        MscStepLimitAlgorithm::size_};
```

Comment:
I might refactor this a bit and add a separate struct for the particle-dependent scalars (since we'll also need muon/hadron `min_range`, `max_step_over_range`, `lowest_electron_energy`) and an accessor in `PhysicsTrackView` to return either the e-/e+ or mu/had scalars depending on the particle ID.